pytentiostat/operator.py

```diff
@@ -1,368 +1,313 @@
-            average_number,
-            line,
-            time_step,
-            conversion_factor,
-            shunt_resistor,
-            config_data,
-            Times,
-            Voltages,
-            Currents
-        )
-
-        return Times, Voltages, Currents, interrupt
-
-    elif exp_type == "CA":
-
-        start_time = start_exp(d9, normalized_voltage, config_data)
-
-        Times, Voltages, Currents, interrupt = read_write(
-            start_time,
-            *pin_objects,
-            step_number,
-            steps_list,
-            time_for_range,
-            average_number,
-            line,
-            time_step,
-            conversion_factor,
-            shunt_resistor,
-            config_data,
-            Times,
-            Voltages,
-            Currents
-        )
-
-        return Times, Voltages, Currents, interrupt
-
-    elif exp_type == "CV":
-
-        start_time = start_exp(d9, normalized_start, config_data)
-        for i in range(cycle_number):
-            Times, Voltages, Currents, interrupt = read_write(
-                start_time,
-                *pin_objects,
-                step_number,
-                first_steps_list,
-                first_time_range,
-                average_number,
-                line,
-                time_step,
-                conversion_factor,
-                shunt_resistor,
-                config_data,
-                Times,
-                Voltages,
-                Currents
-            )
-            Times, Voltages, Currents, interrupt = read_write(
-                start_time,
-                *pin_objects,
-                step_number,
-                second_steps_list,
-                second_time_range,
-                average_number,
-                line,
-                time_step,
-                conversion_factor,
-                shunt_resistor,
-                config_data,
-                Times,
-                Voltages,
-                Currents
-            )
-            Times, Voltages, Currents, interrupt = read_write(
-                start_time,
-                *pin_objects,
-                step_number,
-                third_steps_list,
-                third_time_range,
-                average_number,
-                line,
-                time_step,
-                conversion_factor,
-                shunt_resistor,
-                config_data,
-                Times,
-                Voltages,
-                Currents
-            )
-            i = i+1
-
-        return Times, Voltages, Currents, interrupt
+# Standard Libraries
+import time
+import sys
+import numpy as np
+import signal
+
+# Pytentiostat function files
+from pytentiostat.plotter import plot_initializer, plot_updater
+import pytentiostat.config_reader as cr
+
+Interrupt = False
+Exp_running = False
+
+def signal_handler(signum, frame):
+    """
+    Signal handler is called by signal.signal() when Ctrl+c is pressed by the user.
```

Comment:
first line should be <80 summary of goal of the function, followed by a blank line, then something that looks like this text block (more nice details).  e.g., first line could be:

Changes behavior of keyboard interrupt when experiment is running

This is an internal function and we would probably make it private (not easily visible to users) with a preceding underscore,  so ``_signal_handler()``